timemachine/cpp/src/tibd_exchange_move.cu

```diff
@@ -73,15 +76,27 @@ TIBDExchangeMove<RealType>::TIBDExchangeMove(
         this->num_target_mols_, d_identify_indices_.data, 0);
     gpuErrchk(cudaPeekAtLastError());
 
+    size_t flagged_bytes = 0;
     // Setup buffer for doing the flagged partition
     gpuErrchk(cub::DevicePartition::Flagged(
         nullptr,
-        temp_storage_bytes_,
+        flagged_bytes,
```

Comment:
Not relevant for this PR, but do we need to re-run flagged partition if we accept a move during a batch of moves? (we can toggle the state easily, but we still need to re-partition). If we do, is this a stable partitioning algorithm (document)?

Reply:
We do need to re-run the partition. This is to ensure bit wise determinism between step sizes, it is a stable partitioning algorithm. Will link to the CUB documentation.

Reply:
Added comment in https://github.com/proteneer/timemachine/pull/1235/commits/34147fb6a3b79629ae5a7f3354f89b5e0252ca30 

We could probably get rid of this, but would prioritize batching over making this deterministic.